docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md

```diff
@@ -4,6 +4,10 @@ description: Octopus resources created in cloud providers can also be modeled in
 position: 100
 ---
 
+:::hint
+Octopus version 2022.1 has support for discovering and cleaning up supported types of deployment targets using tags on cloud resources, please see our documentation on [Cloud Target Discovery](/docs/infrastructure/deployment-targets/cloud-target-discovery/index.md) for more information.
```

Comment:
Should this hint also expand on our recommendation made [here](https://github.com/OctopusDeploy/docs/pull/1448/files#diff-3dbf529eab1cf9b9fe14638f80e58b3eb35b1bb9729aaa734d1c7b360173db87R13) e.g.

> We recommend cloud target discovery over the existing [dynamic infrastructure](/docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md) functionality.

Reply:
Hmmm good thought, although given the feature is EAP at the moment I wonder if we should make strong recommendations on it's use over existing features at all? Do you have any thoughts on this @andrewabest ?

Reply:
This wasn't a big deal for me either way. So happy with whatever you decide 😃